src/Git.hs

```diff
@@ -78,7 +79,7 @@ import Configuration (UserConfiguration)
 import Format (format)
 
 import qualified Configuration as Config
-import GitHub.Internal.Prelude (Generic)
+import GitHub.Internal.Prelude (Generic, IsString)
```

Comment:
This seems like a weird place to import them from. Aren't those just base library things?